app/javascript/lib/fetchHelper.js

```diff
@@ -0,0 +1,8 @@
+export const fetchHelper = (response) => {
+  if(!response){
+    return []
+  }
+  return response.results.map(img => {
```

Comment:
what if there's no `response.results`?

Reply:
wouldnt that be covered by !response? 
or should i just add !response | | reponse = ""

Reply:
this is the same situation as above... maybe we should put in the if `!response || !response.results`